Dockerfile

```diff
@@ -11,22 +11,25 @@ RUN yarn --frozen-lockfile && \
 
 # remove development dependencies
 RUN yarn install --production --ignore-scripts --prefer-offline
-RUN yarn cache clean
+
+RUN mkdir -p /artifact && \
+  mv /app/dist /artifact && \
+  mv /app/.next /artifact && \
+  mv /app/node_modules /artifact
+
 
 FROM node:12.18-alpine
 
+RUN apk add --no-cache tzdata
+
 ARG COMMIT_TAG
 ENV COMMIT_TAG=${COMMIT_TAG}
 
-RUN apk add tzdata
-
-COPY . /app
+COPY [^src]* /app/
```

Comment:
Whats the point of doing this if we don't do it for the `server` directory also?